docs/current_docs/manuals/developer/debugging.mdx

```diff
@@ -40,19 +54,19 @@ Using the command `dagger call --debug echo --msg="hello"`, should print
 something like:
 
 ```
-✔ MyModule.echo(msg: "hello"): String! 0.5s
+✔ Mymodule.echo(msg: "hello"): String! 0.5s
```

Comment:
@helderco this is not the current output. The current output contains a very large number of spans, and I wasn't able to find anything similar to what is shown here. Please suggest what should be done for this output.